src/FSharp.Data.GraphQL.Server/Execution.fs

```diff
@@ -500,36 +500,36 @@ let private executeQueryOrMutation (resultSet: (string * ExecutionInfo) []) (ctx
             let! res = 
                 match List.tail path, tree' with
                 | [], t -> 
-                    // printfn "%A\n\n%A\n\n%A" d fieldCtx t
-                    // printfn "Done"
+                    printfn "%A\n\n%A\n\n%A" d fieldCtx t
```

Comment:
You've left print statements uncommented and merged this PR anyway...

Reply:
Actually - I think this was removed.

https://github.com/fsprojects/FSharp.Data.GraphQL/blob/dev/src/FSharp.Data.GraphQL.Server/Execution.fs#L497